Donut/templates/layout.html

```diff
@@ -1,31 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>Donut</title>
-    <link rel="stylesheet" type="text/css" 
-          href="{{ url_for('static', filename='css/style.css') }}" />
-    <!-- Google Fonts -->
-    <link href='http://fonts.googleapis.com/css?family=EB+Garamond|Roboto' rel='stylesheet' type='text/css'>
-    {% block head %}
-    {% endblock head %}
-  </head>
-  <center>
-  <body>
-    {% include "includes/header.html" %}
-    <div id="content">
-      <div id="content-body">
-        {% with messages = get_flashed_messages() %}
-          {% if messages %}
-            {% for message in messages %}
-              <div class="message">{{ message }}</div>
-            {% endfor %}
-          {% endif %}
-        {% endwith %}
-        {% block body %}
-        {% endblock body %}
-      </div>
-    </div>
-    {% include "includes/footer.html" %}
-  </body>
-  </center>
+    <head>
+        <title>Donut</title>
+        
+        <link rel="stylesheet" type="text/css" 
+              href="{{ url_for('static', filename='css/style.css') }}" />
+        <!-- Google Fonts -->
```

Comment:
You can use `{# ... #}` for comments as well. The difference is that HTML style comments are passed on to the user when the page loads, while `{# ... #}` are simply ignored by the template engine so they appear in source code but aren't sent with every request. Minor optimization, but you really don't need to send comments as part of a page's HTML.